frontend/src/metabase/visualizations/echarts/cartesian/model/axis.ts

```diff
@@ -399,6 +399,10 @@ export const getYAxisLabel = (
     return undefined;
   }
 
+  if (settings.series_settings?.[axisColumn.name]?.title) {
+    return settings.series_settings[axisColumn.name].title;
```

Comment:
The key there is `field_ref` as far as I remember and it may change with MLv2. I think we may want to change `getYAxisModel` so that it accepts not `seriesKeys` but `seriesModels` for an axis. Then in `getYAxisLabel(seriesModels, column, settings);` we will be able to grab the title of the first seriesModel when needed

Reply:
AFAICT, we're using `column.name` there:

![CleanShot 2024-03-21 at 11 24 35@2x](https://github.com/metabase/metabase/assets/17258145/b4ba385f-3677-4514-bdad-8ffa24bfcf39)

Reply:
> I think we may want to change `getYAxisModel` so that it accepts not `seriesKeys` but `seriesModels` for an axis. Then in `getYAxisLabel(seriesModels, column, settings);` we will be able to grab the title of the first seriesModel when needed

It seems like we'd need to change a bunch of other functions for that as well, I'll take a stab in another PR. Thanks for pointing me in that direction!